src/app/modules/consultations/consultation-profile/read-respond/read-respond.component.ts

```diff
@@ -48,9 +50,10 @@ export class ReadRespondComponent implements OnInit {
   };
 
   languages = [
-    { id: 'en', name: 'English' },
-    { id: 'hi', name: 'Hindi' },
-    { id: "od", name: "Odia" },
+    { id: LANGUAGE_IDS.ENGLISH, name: "English" },
+    { id: LANGUAGE_IDS.HINDI, name: "Hindi" },
+    { id: LANGUAGE_IDS.ODIA, name: "Odia" },
+    { id: LANGUAGE_IDS.MARATHI, name: "Marathi" },
```

Comment:
This object is being duplicated multiple times in the codebase. Since it is a static object, it can be defined in the same constants file